### DISCUSSION TITLE
Should nested GraphQL filters still work?

### DISCUSSION THREAD

**Discussion by Jakob-em** [directus/directus#4889] [Archive]
I just updated to the latest rc version (from 34 to 54) and now nested graphql filtering is no longer possible.
For example a query like this:
```
query {
    collection1 {
        collection2(filter: ...) {

        }
    }
}

```
worked with rc-34 but doesn't work with rc-54.

Was the syntax changed with the recent GraphQL related changes or is this a bug?
I would be happy to work on a PR if this way of using GraphQL should still work.

**Comment by rijkvanzanten** [comment]
> Was the syntax changed with the recent GraphQL related changes or is this a bug?

The latter! I'm only adding the args to the resolver on the item itself, but not retroactively on the nested collection fields 👍🏻

**Reply by Jakob-em** [reply]
Okay I will try to submit a PR for this.
Thanks for the quick response!

**Reply by rijkvanzanten** [reply]
Don't worry about a PR, I'm about to push up the fix 🙂
